src/controllers/index.ts

```diff
@@ -0,0 +1,15 @@
+import { Logger } from 'embark-logger';
+import { Environment } from '../types';
+import Client from '../client';
+
+export default abstract class Controller {
+  protected client: Client;
+  constructor(env: Environment, protected logger: Logger) {
+    this.client = new Client(env);
+  }
+
+  protected async login() {
```

Comment:
Isn't this already part of the login controller?

Reply:
Good point! Removed the unneeded login controller.